client/landing/gutenboarding/onboarding-block/design-selector/style.scss

```diff
@@ -1,4 +1,22 @@
 .design-selector {
+	h1 {
+		color: #23282d;
+		font-size: 2em;
+		font-weight: bold;
+		margin-bottom: 0.25em;
+		text-align: center;
+	}
+
+	h2,
+	h3 {
+		color: #23282d;
+		font-size: 1.3em;
+		margin-bottom: 3em;
+		text-align: center;
+	}
+}
```

Comment:
I'd prefer adding a specific classname for styling. It seems like using generic elements (even scoped like this) tends to come back to bite you.

Reply:
Okay, can do